main.cpp

```diff
@@ -131,28 +211,59 @@ int main()
     //example of calling that function, storing the value, and passing it to ignoreUnused at the end of main()
     auto carRented = rentACar(6, 2); 
     
-    //1)
+    //1)
+
+    auto planeFlown = flyAPlane(900, 5, 25);
     
-    //2)
+    //2)
+
+    auto average = calculateAverage(4, 5, 7);
     
-    //3)
+    //3)
+
+    auto dishesWashed = washDishes(true, 34);
     
-    //4)
+    //4)
+
+    auto guitarPlayed = playGuitar(true, 6, 2);
     
-    //5)
+    //5)
+
+    auto candyEaten = eatCandy(65, false);
     
-    //6)
+    //6)
+
+    auto chordsNeeded = numberOfChordsNeeded(6, false);
     
-    //7)
+    //7)
+
+    auto pluginCreated = createPlugin(5006, 7.746f);
     
-    //8)
+    //8)
+
+    auto scoreCardCreated = createScoreCard('b', 7);
     
-    //9)
+    //9)
+
+    auto goneBirding = goBirding(56, 87);
     
-    //10)
+    //10)
+
+    auto madePizza = makeAPizza(4, true);
     
     
-    ignoreUnused(carRented);
+    ignoreUnused(carRented);
+    ignoreUnused(planeFlown);
+    ignoreUnused(average);
+    ignoreUnused(dishesWashed);
+    ignoreUnused(guitarPlayed);
+    ignoreUnused(candyEaten);
+    ignoreUnused(chordsNeeded);
+    ignoreUnused(pluginCreated);
+    ignoreUnused(scoreCardCreated);
+    ignoreUnused(goneBirding);
+    ignoreUnused(madePizza);
+

```

Comment:
Call `ignoreUnused()` once, and pass all of your variables to it, just like you did on line 74
- [ ] revise